libsrc/time/tvc/clock.asm

```diff
@@ -0,0 +1,24 @@
+;
+;   Videoton TV Computer C stub
+;   Sandor Vass - 2019
+;   Based on the source of
+;
+;	Enterprise 64/128 clock()
+;
+;	stefano 5/4/2007
+;
+
+        SECTION code_clib
+        PUBLIC	clock
+        PUBLIC	_clock
+        include "target/tvc/def/tvc.def"
+.clock
+._clock
+;    INTINC is 0b1dh
+    di
```

Comment:
Please use `call l_push_di` and `call l_pop_ei` - these functions preserve the interrupt enabled flag - i.e. `l_pop_ei` won't enable interrupts if they were disabled when `l_push_di` was called.

Reply:
As a matter of fact there is no need to do ei/di as this counter is incremented in the TVC's interrupt routine. I fix this too by eliminating the ei/di pairs.